### ISSUE TITLE
Add a ClassEncoder and ClassDecoder primitives

### ISSUE THREAD

**Issue by csala** [HDI-Project/MLPrimitives#24]
Encoding and Decoding functionality should be added in a way that allows converting `y` classes to integers at the beginning of the pipeline and back to strings at the end.

This can be done using the [scikit-learn LabelEncoder](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html).

For this we need:
* A ClassEncoder class with two methods:
    * `fit(y)`:
        * Creates an instance of a LabelEncoder
        * calls fit on y
    * `encode(y)`:
        * Calls the LabelEncoder `transform` method and returns two things:
            * The transformed y
            * The LabelEncoder `classes_` attribute, which will be called `classes` in the Annotation
* A ClassDecoder class with two methods:
    * `fit(classes)`:
        * Creates an instance of a LabelEncoder
        * assigns the `classes` variable to the `LabelEncoder.classes_` attribute.
    * `decode(y)`:
        * calls the label encoder `inverse_transform` method on y and returns its output.
